src/models/StyleTags.js

```diff
@@ -186,7 +186,7 @@ const makeSpeedyTag = (
     const size = sizes[marker]
     const sheet = sheetForTag(el)
     const removalIndex = addUpUntilIndex(sizes, marker)
-    deleteRules(sheet, removalIndex, size)
+    deleteRules(sheet, removalIndex - 1, size)
```

Comment:
Let's double check that this is still necessary, feel like this could cause all sorts of weird bugs.

Reply:
It was necessary to remove that one (index out of bounds)  bug for global styles. But I am not sure what other bugs this might cause. 🤔

Reply:
Hmmm. Could be. I will need to see what's going on internally. But shouldn't more tests fail if that's the case? Can you think of a test case that I can write to test around this scenario?